spec/kibana_spec.rb

```diff
@@ -1,24 +1,9 @@
 # encoding: utf-8
 require 'spec_helper'
 
-kibana_version = '4.4.1'
```

Comment:
@conorsch Okay, I added it back and used the package resource type. I'm not great at Serverspec, and can't find a good example in the docs offhand, so maybe you can tell me how to do this. It would seem we need `should_be_installed.with_version(kibana_version)` but I don't know how to express "greater than", and not sure how to evaluate the var (does it need to be in enclosed in `#{}`?) either.